src/main/webapp/WEB-INF/views/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head lang="en">
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" type="text/css" href="login-style.css">
+    <link rel="stylesheet" type="text/css" th:href="@{/css/login.css}">
+</head>
+<body>
+<div class="login-page">
+    <div class="form">
+        <!--<form class="" method="post" th:action="@{/j_spring_security_check}">
```

Comment:
Do you need this code?

Reply:
This code was needed only when I created a page.